### ISSUE TITLE
AttributeError: 'Sphinx' object has no attribute 'warn'

### ISSUE THREAD

**Issue by honzajavorek** [djungelorm/sphinx-tabs#39]
When building on ReadTheDocs with Sphinx v2.0.1 I'm suddenly experiencing following, which breaks the build. I was not able to reproduce locally immediately, but the problem seams to be clear from the stack trace.

```
Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/cojeapi/envs/latest/lib/python3.6/site-packages/sphinx_tabs/tabs.py", line 301, in copy_assets
    app.warn(
AttributeError: 'Sphinx' object has no attribute 'warn'
```

If I find time and I'm able to reproduce locally as well, I'll try to send a PR.

**Comment by honzajavorek** [comment]
I made a PR to fix this. I believe there is a workaround as well: set `sphinx_tabs_nowarn` setting to `True`.

**Comment by djungelorm** [comment]
Fixed in #43
